mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -1134,19 +1148,18 @@ class WidgetAdapter(
             boundItem = widget.item
         }
 
-        override fun handleRowClick() {
+        // Up and down buttons callback
+        override fun invoke(v: View, value: String?) {
+            value?.let { connection.httpClient.sendItemCommand(boundItem, value) }
+        }
```

Comment:
I'd prefer to have this one as a member variable instead, since the method name is somewhat unspecific (and yes, I realize there's a comment, but still :-) ).